userguide/sharing.rst

```diff
@@ -34,13 +34,15 @@ system to meet the requirements of the network.
 
 These types of shares and services are available:
 
-* :ref:`AFP <Apple (AFP) Shares>`: Apple File Protocol shares are
-  often used when the client computers all run macOS. Apple has
-  slowly shifted to preferring :ref:`SMB <Windows (SMB) Shares>` for
-  modern networks, although Time Machine still requires AFP.
+* :ref:`AFP <Apple (AFP) Shares>`: Apple Filing Protocol shares are
+  used when the client computers all run macOS. Apple has deprecated
```

Comment:
This needs a qualifier, like "often", or maybe "only".  Even that is too absolute.  Something like "...are generally used only when legacy client computers all run macOS."

Reply:
I've reworked this whole section.